addons/main/script_macros.hpp

```diff
@@ -37,7 +37,7 @@
 #define GETGVAR(var1,var2) GETMVAR(GVAR(var1),var2)
 #define GETEGVAR(var1,var2,var3) GETMVAR(EGVAR(var1,var2),var3)
 
-#define ARR_SELECT(ARRAY,INDEX,DEFAULT) if (count ARRAY > INDEX) then {ARRAY select INDEX} else {DEFAULT}
+#define ARR_SELECT(ARRAY,INDEX,DEFAULT) (if (count ARRAY > INDEX) then {ARRAY select INDEX} else {DEFAULT})
```

Comment:
```sqf
(ARRAY param [INDEX, DEFAULT])
```

Reply:
^ Just an idea.

Reply:
Could also remove `ARR_SELECT` which only has 1 use:
https://github.com/acemod/ACE3/search?utf8=%E2%9C%93&q=ARR_SELECT&type=

Reply:
what about BWC?

Reply:
ACE's macros aren't supposed to be used by others, specially not one this archaic.

Reply:
Actually they are: https://github.com/acemod/arma-project-template/blob/master/addons/main/script_macros.hpp

But this one can safely be removed as it plays no role anymore.

Reply:
I mean, it can also stay as is. Doesn't have to be removed with this PR. I'm for merging this PR.

Reply:
OK, will not update it